lib/main/traceLogger.ts

```diff
@@ -0,0 +1,212 @@
+import log from 'electron-log'
```

Comment:
Should we use something like opentelemetry instead of making our own trace logger?

One of the issues I see with this is we'll have to figure out how to pipe logs to cloudwatch, and we'll have to add to the metadata for grpc requests. If we use a more mature library that handles all of that, it could simplify a lot of the headache around logging https://www.npmjs.com/package/@opentelemetry/instrumentation-grpc